frontend/src/helpers/dataOverlay.js

```diff
@@ -0,0 +1,97 @@
+const DEFAULT_GENE_COLOR = '#feb';
+const DEFAULT_METABOLITE_COLOR = '#9df';
+
+const DATA_TYPES_COMPONENTS = {
+  transcriptomics: {
+    componentType: 'gene',
+    className: 'enz',
+    defaultColor: DEFAULT_GENE_COLOR,
+  },
+  metabolomics: {
+    componentType: 'metabolite',
+    className: 'met',
+    defaultColor: DEFAULT_METABOLITE_COLOR,
+  },
+};
+
+/*
+ * @param { Blob } file
+ *
+ * @example
+ * try {
+ *   const data = await parseCustomFile(file);
+ * } catch (e) {
+ *   console.error(e)
+ * }
+ */
```

Comment:
I haven't seen this type of comment in other places in our codebase.

Reply:
Fixed with commit 703c572b298e278bac79a284a932c68697268188